scalafix-tests/input/src/main/scala/test/ExplicitReturnTypes.scala

```diff
@@ -1,9 +1,21 @@
 /*
 rewrites = ExplicitReturnTypes
+explicitReturnTypes.memberKind = [Def, Var]
+explicitReturnTypes.memberVisibility = [Public, Protected]
  */
 package test
 
 object ExplicitReturnTypes {
+  val a = 1
+  def b() = List
+  var c = true
+  protected val d = 1.0f
```

Comment:
I think intellij-scala excludes cases when the right-hand side is a plain literal, see https://blog.jetbrains.com/scala/2016/10/05/beyond-code-style/

We could do something similar, have an option to skip the rewrite if the right-hand side body is a `Lit`.

```
explicitReturnTypes.skipSimpleDefinitions: Boolean // true by default
```

We can later expand on the heuristic to determine if a definition is "simple" or not.